pypesto/objective/amici_objective.py

```diff
@@ -622,13 +616,15 @@ def add_sim_grad_to_opt_grad(par_opt_ids,
         Coefficient for sim_grad when adding to opt_grad.
     """
 
-    for par_sim_idx, par_opt_id in enumerate(mapping_par_opt_to_par_sim):
+    par_sim_idx = 0
+    for par_opt_id in mapping_par_opt_to_par_sim:
```

Comment:
could you add a comment here that we need to leave non-string overrides out of the index, as sim_grad contains only values for str overrides?
will help maintain the code later.